sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -28,11 +30,21 @@ public class CommunicationTestEnvironment : TestEnvironment
 
         public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
 
+        public string SkipSmsTest => GetOptionalVariable(SkipIntSmsTestEnvironmentVariableName) ?? "False";
```

Comment:
Does it make sense to do this. 
`public string SkipSmsTest => bool.Parse(GetOptionalVariable(SkipIntSmsTestEnvironmentVariableName)) ?? false;`

Reply:
I think bool.Parse(null) will throw an exception
https://docs.microsoft.com/en-us/dotnet/api/system.boolean.parse?view=net-5.0